extensions/in-memory/assetindex-memory/src/main/java/org/eclipse/dataspaceconnector/metadata/memory/InMemoryAssetIndex.java

```diff
@@ -68,7 +68,8 @@ public Asset findById(String assetId) {
         return assets.isEmpty() ? null : assets.get(0);
     }
 
-    void add(Asset asset, DataAddress address) {
+    // TODO: address is not used and it should be deleted
+    public void add(Asset asset, DataAddress address) {
```

Comment:
feel free to do so, but we need to touch that again in [this issue](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/172) anyway